source/extensions/filters/http/jwt_authn/filter_config.cc

```diff
@@ -1,9 +1,17 @@
 #include "extensions/filters/http/jwt_authn/filter_config.h"
 
+#include "common/common/empty_string.h"
+
 namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace JwtAuthn {
+namespace {
+
+// The maximum size for this variable.
+constexpr size_t TopRequirementNameForDebugSize = 100;
```

Comment:
Specify the size is for the number of characters. Otherwise it seems like it's limiting the number of elements in a list.

TopRequirementNameForDebugMaxChars

Reply:
Done